hawkbit-runtime/hawkbit-update-server/src/main/resources/application.properties

```diff
@@ -8,8 +8,9 @@
 #
 
 # User Security
-security.user.name=admin
-security.user.password=admin
+spring.security.user.name=admin
+spring.security.user.password={noop}admin
+spring.main.allow-bean-definition-overriding=true
```

Comment:
I think the properties below (line 31ff) starting with a "spring.http" prefix need to be renamed to "sping.servlet"